07-routing-gossip.md

```diff
@@ -19,16 +19,31 @@ It contains the necessary signatures by the sender to construct the `channel_ann
 
 1. type: 259 (`announcement_signatures`)
 2. data:
+    * [32:funding-txid]
+    * [1:funding-output-index]
     * [8:channel-id]
     * [64:node-signature]
     * [64:bitcoin-signature]
 
 The willingness of the endpoints to announce the channel is signaled during the connection setup by setting a `channels_public` bit in the `localfeatures` field.
-Should both endpoints have signaled that they'd like to publish the channel then the `announcement_signatures` message MUST directly sent following the `funding_locked` message that established the corresponding channel.
+
+### Requirements
+
+If both endpoints have signaled that they'd like to publish the channel then the `announcement_signatures` message MUST be sent, otherwise they MUST NOT be sent.
+
+If sent, `announcement_signatures` messages MUST NOT be sent until `funding_locked` has been sent, and the funding transaction is has at least 6 confirmations.
```

Comment:
Yes, I think we need to detach this now we accept that the short channel-id can change on an active channel.

I didn't want to do it before because we didn't know when to re-transmit it, but I think the new language fixes this.